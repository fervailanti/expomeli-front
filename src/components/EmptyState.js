import React from 'react'
import { images } from '../images'

const EmptyState = () => (
  <div className="flex flex-col md:flex-row items-center justify-center p-5 md:p-10">
    <img src={images.emptySearchIcon} className="h-20 md:mr-10" alt="empty-search" />
    <div>
      <p className="text-2xl mt-5 md:mt-0">¡Ups! No hay resultados para tu búsqueda.</p>
      <ul className="list-disc list-inside mt-3 md:mt-2 text-md font-thin">
        <li className="mb-2 md:mb-0">
          <b>Revisá la ortografía</b> de la palabra.
        </li>
        <li className="mb-2 md:mb-0">
          Utilizá <b>menos palabras</b> o <b>más genéricas</b>.
        </li>
        <li className="mb-2 md:mb-0">
          Revisá tu <b>conexión a internet</b>.
        </li>
      </ul>
    </div>
  </div>
)

export default EmptyState
