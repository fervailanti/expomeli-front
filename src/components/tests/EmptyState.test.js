import React from 'react'
import EmptyState from '../EmptyState'
import { render } from '@testing-library/react'

it('EmptyState component should render correctly', () => {
  const tree = render(<EmptyState />)
  expect(tree.container).toMatchSnapshot()
})
