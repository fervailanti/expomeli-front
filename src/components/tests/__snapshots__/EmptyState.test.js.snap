// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState component should render correctly 1`] = `
<div>
  <div
    class="flex flex-col md:flex-row items-center justify-center p-5 md:p-10"
  >
    <img
      alt="empty-search"
      class="h-20 md:mr-10"
      src="[object Object]"
    />
    <div>
      <p
        class="text-2xl mt-5 md:mt-0"
      >
        ¡Ups! No hay resultados para tu búsqueda.
      </p>
      <ul
        class="list-disc list-inside mt-3 md:mt-2 text-md font-thin"
      >
        <li
          class="mb-2 md:mb-0"
        >
          <b>
            Revisá la ortografía
          </b>
           de la palabra.
        </li>
        <li
          class="mb-2 md:mb-0"
        >
          Utilizá 
          <b>
            menos palabras
          </b>
           o 
          <b>
            más genéricas
          </b>
          .
        </li>
        <li
          class="mb-2 md:mb-0"
        >
          Revisá tu 
          <b>
            conexión a internet
          </b>
          .
        </li>
      </ul>
    </div>
  </div>
</div>
`;
